tools/cldr-apps/js/src/views/GenerateVxml.vue

```diff
@@ -84,7 +84,9 @@ defineExpose({
 </script>
 
 <template>
-  <div v-if="!hasPermission">Please log in as Admin to use this feature.</div>
+  <div v-if="!hasPermission">
+    Please log in as a user with sufficient permissions.
```

Comment:
this will practically never happen, since the menu doesn't present the option if the user lacks permission